src/m365/planner/commands/roster/roster-add.ts

```diff
@@ -0,0 +1,49 @@
+import { Logger } from '../../../../cli/Logger';
+import request, { CliRequestOptions } from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+
+class PlannerRosterAddCommand extends GraphCommand {
+  public get name(): string {
+    return commands.ROSTER_ADD;
+  }
+
+  public get description(): string {
+    return 'Creates a new Microsoft Planner Roster';
+  }
+
+  constructor() {
+    super();
+  }
+
+  public async commandAction(logger: Logger): Promise<void> {
+    if (this.verbose) {
+      logger.logToStderr('Creating a new Microsoft Planner Roster');
+    }
+
+    try {
+      const requestBody: any = {
+        "@odata.type": "#microsoft.graph.plannerRoster"
```

Comment:
Since we are using no metadata, we don't have to specify a type. I found out that we just have to provide an empty body and that'll work.